feature-settings-impl/src/main/java/io/novafoundation/nova/feature_settings_impl/presentation/cloudBackup/BackupSettingsViewModel.kt

```diff
@@ -90,7 +110,26 @@ class CloudBackupSettingsViewModel(
     }
 
     fun cloudBackupManageClicked() {
-        TODO()
+        when (syncedState.value) {
+            BackupSyncOutcome.Ok,
+            BackupSyncOutcome.StorageAuthFailed -> return
+
+            BackupSyncOutcome.UnknownPassword,
+            BackupSyncOutcome.CorruptedBackup,
+            BackupSyncOutcome.OtherStorageIssue -> {
+                listSelectorMixin.showSelector(
+                    R.string.manage_cloud_backup,
+                    listOf(manageBackupDeleteBackupItem())
+                )
+            }
+
+            else -> {
```

Comment:
I suggest to avoid else branch to let compiler help us when we introduce changes to `BackupSyncOutcome` variants